### ISSUE TITLE
clang 3.4 broken

### ISSUE THREAD

**Issue by henryiii** [silkeh/docker-clang#7]
The following recipe:

```bash
docker run --rm -it silkeh/clang:3.4
mkdir cmake_test
cd cmake_test
printf "cmake_minimum_required(VERSION 3.1)\n project(MyProject LANGUAGES CXX)" > CMakeLists.txt
cmake .
```

broke recently (probably about a month ago):

```
-- The CXX compiler identification is Clang 3.4.2
-- Check for working CXX compiler: /usr/local/bin/clang++
-- Check for working CXX compiler: /usr/local/bin/clang++ -- broken
CMake Error at /usr/share/cmake-3.13/Modules/CMakeTestCXXCompiler.cmake:45 (message):
  The C++ compiler

    "/usr/local/bin/clang++"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /cmake_test/CMakeFiles/CMakeTmp

    Run Build Command:"/usr/bin/make" "cmTC_f4363/fast"
    /usr/bin/make -f CMakeFiles/cmTC_f4363.dir/build.make CMakeFiles/cmTC_f4363.dir/build
    make[1]: Entering directory '/cmake_test/CMakeFiles/CMakeTmp'
    Building CXX object CMakeFiles/cmTC_f4363.dir/testCXXCompiler.cxx.o
    /usr/local/bin/clang++     -o CMakeFiles/cmTC_f4363.dir/testCXXCompiler.cxx.o -c /cmake_test/CMakeFiles/CMakeTmp/testCXXCompiler.cxx
    Linking CXX executable cmTC_f4363
    /usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_f4363.dir/link.txt --verbose=1
    /usr/local/bin/clang++       -rdynamic CMakeFiles/cmTC_f4363.dir/testCXXCompiler.cxx.o  -o cmTC_f4363
    /usr/bin/ld: cannot find crtbegin.o: No such file or directory
    /usr/bin/ld: cannot find -lstdc++
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    make[1]: *** [CMakeFiles/cmTC_f4363.dir/build.make:87: cmTC_f4363] Error 1
    make[1]: Leaving directory '/cmake_test/CMakeFiles/CMakeTmp'
    make: *** [Makefile:121: cmTC_f4363/fast] Error 2




  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:2 (project)


-- Configuring incomplete, errors occurred!
See also "/cmake_test/CMakeFiles/CMakeOutput.log".
See also "/cmake_test/CMakeFiles/CMakeError.log".
```

(MWE, an actual example is [CLI11](https://github.com/CLIUtils/CLI11)'s CI system). Also fails with 3.5, works with (larger?) 8. image.

Edit: Could it be that clang 3.x is not supported by libstdc++ 8? https://github.com/silkeh/docker-clang/commit/16ae1f46dcf5c8b0c4066ad2c958d03a1ad9b207

Edit: Shorter version for easier version checking:
```bash
docker run --rm silkeh/clang:3.4 /bin/bash -c 'mkdir cmake_test; cd cmake_test; printf "cmake_minimum_required(VERSION 3.1)\n project(MyProject LANGUAGES CXX)" > CMakeLists.txt; cmake .'
```

**Comment by henryiii** [comment]
Yes, forcibly downgrading these two packages to version 7 fixes the compile. Running the command above on each image shows that versions up to and including 3.8 are now broken, while 3.9 and above work. My recommendation is to revert the 3.3-3.8 versions back to `libstdc++ 7`.

**Comment by silkeh** [comment]
I have merged #8. The automated builds should have started and new images should be available within a few hours.
Closing this issue for now, but don't hesitate to reopen it if the issue isn't fully resolved (or pops up again).

**Comment by henryiii** [comment]
Works perfectly again, thank you!
